Bleedify/Bleedify.Tests/PersonalTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using BleedifyModels.ModelsEF;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Bleedify.Tests
+{
+    [TestClass]
+    public class PersonalTests
+    {
+        private Personal _testPersonal;
+
+
+        [TestInitialize]
+        public void InitializeTest()
+        {
+            _testPersonal = new Personal();
+        }
+
+
+        [TestMethod]
+        public void TestGettersPersonal()
+        {
+            // Arrange
```

Comment:
Nu-s obligatorii comentariile cu cele 3 actiuni dintr-un test. Dar apreciez ca le-ai pus.

Reply:
Also, daca tot vrei sa le pui, ar fi bine sa fii consistent si sa le pui peste tot dar in loc de // mai bine pui #region si #endregion